### ISSUE TITLE
driver failure: native driver is not running

### ISSUE THREAD

**Issue by smacker** [bblfsh/sdk#380]
Try to parse file from this issue: https://github.com/bblfsh/python-driver/issues/182

The driver is broken after it and doesn't get restarted. For example, it is broken right now on https://play.bblf.sh/ and parsing of python files doesn't work anymore.

**Comment by bzz** [comment]
@dennwc as soon as https://github.com/bblfsh/bblfshd/releases/tag/v2.12.0 is released, here are results using it

```
$ docker run -it --name bblfshd --privileged -p 9432:9432 bblfsh/bblfshd:v2.12.0-drivers --log-level=debug

$ wget https://gist.githubusercontent.com/smacker/7a618e37997b7ed9d3b688bea04561f1/raw/3448e22a1b884f82fc513f9cc3d2334af4544dba/decoder.py
$ bblfsh-cli decoder.py
driver failure: invalid character 'I' looking for beginning of value

$ docker exec bblfshd bblfshctl status
+----------+----------------+---------------+---------+--------+
| LANGUAGE | SUCCESS/FAILED | STATE/DESIRED | WAITING | EXITED |
+----------+----------------+---------------+---------+--------+
| python   | 0/1            | 1/1           | 0       | 0      |
+----------+----------------+---------------+---------+--------+
Response time 10.8µs

$ bblfsh-cli ./python-driver/fixtures/while.py
driver failure: native driver is not running

$ docker exec bblfshd bblfshctl status
+----------+----------------+---------------+---------+--------+
| LANGUAGE | SUCCESS/FAILED | STATE/DESIRED | WAITING | EXITED |
+----------+----------------+---------------+---------+--------+
| python   | 0/2            | 1/1           | 0       | 0      |
+----------+----------------+---------------+---------+--------+
Response time 13.7µs
```

TL;DR: latest python driver still hits https://github.com/bblfsh/python-driver/issues/182 and this issue is still reproducible.

Server logs
```
[2019-03-26T23:34:32Z]  INFO server listening in /tmp/rpc.sock (unix) id=01d6y5ny1g0v3hepkbkypcc9gj language=python
[2019-03-26T23:34:33Z]  INFO new driver instance started bblfsh/python-driver:latest (01d6y5ny1g0v3hepkbkypcc9gj)
[2019-03-26T23:34:33Z] ERROR request processed content 31289 bytes error: rpc error: code = Internal desc = invalid character 'I' looking for beginning of value elapsed=1.4061087s filename=decoder.py language=python
[2019-03-26T23:35:13Z] DEBUG detected language "python", filename "while.py"
[2019-03-26T23:35:13Z] ERROR request processed content 80 bytes error: rpc error: code = Internal desc = native driver is not running elapsed=2.0277ms filename=while.py language=python
```

**Comment by dennwc** [comment]
Found a cause, working on a fix.
